lldb/unittests/DAP/CMakeLists.txt

```diff
@@ -10,8 +10,16 @@ add_lldb_unittest(DAPTests
   VariablesTest.cpp
 
   LINK_LIBS
+    liblldb
     lldbDAP
+    lldbUtilityHelpers
     LLVMTestingSupport
   LINK_COMPONENTS
     Support
   )
+
+set(test_inputs
+  linux-x86_64.out
+  linux-x86_64.core
```

Comment:
Could we obj2yaml these and programmatically yaml2obj them again?

Reply:
yaml2obj on core files doesn't really work. It should be possible to do that with minidump files though.

Reply:
Created yaml dumps of both the core and out files.

Reply:
I assume Pavel's talking about the memory just being bytes. I'd say the YAML file is marginally better than the binaries. A minidump would be even better but I don't feel like we should block this PR on that.

Reply:
I used minidump to make the linux-x86_64.core.yaml file (it starts with `--- !minidump`).  I just needed to load the exiting target + core and use `(lldb) process save-core --plugin-name=minidump <path>` to get the file.

Reply:
> I assume Pavel's talking about the memory just being bytes.

No, that's actually the best you can get (without inventing something completely new), even with minidumps. With core files you just get nothing because they don't fit yaml2obj's model of elf files. elf2yaml's main use case is (unlinked) object files produced by the compiler, which have only section headers (no program headers). OTOH, core files consist of *only* program headers, and the tool just doesn't handle that. These days I *think* it's possible to generate a somewhat realistic core file with yaml2obj, but you have to write the yaml by hand. obj2yaml won't generate it for you.

Reply:
TIL, thanks for the explanation.